app/main/controller/presto_controller.py

```diff
@@ -24,12 +24,16 @@ class PrestoResource(Resource):
     @api.doc('Receive a presto callback for a given `model_type`')
     def post(self, action, model_type):
         data = request.args or request.json
-        print(data)
         app.logger.info(f"PrestoResource {action}")
         if action == "add_item":
+            app.logger.info(f"Data looks like {data}")
```

Comment:
Let's move this before the `if` so that we log the content of `data` in all cases?